chain/network/src/peer.rs

```diff
@@ -193,6 +202,8 @@ pub struct Peer {
     peer_counter: Arc<AtomicUsize>,
     /// The last time a Epoch Sync request was received from this peer
     last_time_received_epoch_sync_request: Instant,
+    /// Cache of recently routed messages, this allows us to drop duplicates
+    routed_message_cache: SizedCache<(PeerId, PeerIdOrHash, Signature), DateTime<Utc>>,
```

Comment:
suggestion: use `std::time::Instant` here rather than `DateTime<Utc>`. Or is there some reason why we need calendar (as opposed to monotonic) time here? Calendar time is a pain -- it can go backwards, and you can be tricked into thinking wrong about calendar time...

Reply:
@matklad We use DataTime<Utc> a lot of in our code. I'll switch to use Instant

Reply:
Thanks, fixed.